packages/core/components/TooltipLink/TooltipLink.css

```diff
@@ -1,10 +1,16 @@
 .tooltip-link {
   text-decoration: underline;
-  text-decoration-color: #005ea2;
+  text-decoration-color: #1c1d1f;
   text-decoration-style: dotted;
   text-decoration-thickness: 2px;
   text-underline-offset: 0.25rem;
   position: relative;
   cursor: pointer;
-  color: #005ea2;
+  color: #1c1d1f;
+  font-family: Arial !important;
```

Comment:
Use one of our defined fonts in this file: `packages/core/styles/_global-variables.scss`

```
  --app-font-main: 'Nunito', sans-serif;
  --app-font-secondary: 'Poppins', sans-serif;
  ```

Reply:
Fixed!